### PR TITLE
[Merged by Bors] - docs(GroupTheory/SpecificGroups/Cyclic): change "Cycle" to "Cyclic"

### PR DESCRIPTION
**Pull request by faenuccio**
Two declaration about _cyclic_ groups had "cycle" in their name. This PR rename them.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/GroupTheory/SpecificGroups/Cyclic.lean [hunk 1]
```diff
@@ -490,13 +490,13 @@ theorem commutative_of_cyclic_center_quotient [IsCyclic H] (f : G →* H) (hf :
...
+@[to_additive commutativeOfAddCyclicCenterQuotient
       "A group is commutative if the quotient by the center is cyclic."]
-def commGroupOfCycleCenterQuotient [IsCyclic H] (f : G →* H) (hf : f.ker ≤ center G) :
+def commGroupOfCyclicCenterQuotient [IsCyclic H] (f : G →* H) (hf : f.ker ≤ center G) :
     CommGroup G :=
   { show Group G by infer_instance with mul_comm := commutative_of_cyclic_center_quotient f hf }
-#align comm_group_of_cycle_center_quotient commGroupOfCycleCenterQuotient
-#align commutative_of_add_cycle_center_quotient commutativeOfAddCycleCenterQuotient
+#align comm_group_of_cyclic_center_quotient commGroupOfCyclicCenterQuotient
+#align commutative_of_add_cyclic_center_quotient commutativeOfAddCyclicCenterQuotient
```
Comment (faenuccio) [inline Mathlib/GroupTheory/SpecificGroups/Cyclic.lean:499]:
```suggestion
```

File: Mathlib/GroupTheory/SpecificGroups/Cyclic.lean [hunk 2]
```diff
@@ -464,10 +464,10 @@ open Subgroup
 variable {G : Type*} {H : Type*} [Group G] [Group H]
 
 /-- A group is commutative if the quotient by the center is cyclic.
-  Also see `commGroup_of_cycle_center_quotient` for the `CommGroup` instance. -/
+  Also see `commGroup_of_cyclic_center_quotient` for the `CommGroup` instance. -/
```
Comment (b-mehta) [inline Mathlib/GroupTheory/SpecificGroups/Cyclic.lean:467]:
```suggestion
  Also see `commGroupOfCyclicCenterQuotient` for the `CommGroup` instance. -/
```
I think the old version is a hybrid Lean 3/4 name?

File: Mathlib/GroupTheory/SpecificGroups/Cyclic.lean [hunk 3]
```diff
@@ -490,13 +490,11 @@ theorem commutative_of_cyclic_center_quotient [IsCyclic H] (f : G →* H) (hf :
 #align commutative_of_add_cyclic_center_quotient commutative_of_add_cyclic_center_quotient
 
 /-- A group is commutative if the quotient by the center is cyclic. -/
-@[to_additive commutativeOfAddCycleCenterQuotient
+@[to_additive commutativeOfAddCyclicCenterQuotient
```
Comment (b-mehta) [inline Mathlib/GroupTheory/SpecificGroups/Cyclic.lean:493]:
```suggestion
@[to_additive addCommGroupOfAddCyclicCenterQuotient
```
seems like the correct name here

**Review by b-mehta — COMMENTED** [review]
bors d+

File: Mathlib/GroupTheory/SpecificGroups/Cyclic.lean [hunk 4]
```diff
@@ -515,7 +515,7 @@ variable {G : Type*} {H : Type*} [Group G] [Group H]
   Also see `commGroup_of_cycle_center_quotient` for the `CommGroup` instance. -/
 @[to_additive
       "A group is commutative if the quotient by the center is cyclic.
-      Also see `addCommGroup_of_cycle_center_quotient` for the `AddCommGroup` instance."]
+      Also see `addCommGroup_of_cyclic_center_quotient` for the `AddCommGroup` instance."]
```
Comment (Ruben-VandeVelde) [inline Mathlib/GroupTheory/SpecificGroups/Cyclic.lean:518]:
```suggestion
      Also see `addCommGroupOfCyclicCenterQuotient` for the `AddCommGroup` instance."]
```
no?

File: Mathlib/GroupTheory/SpecificGroups/Cyclic.lean [hunk 5]
```diff
@@ -515,7 +515,7 @@ variable {G : Type*} {H : Type*} [Group G] [Group H]
   Also see `commGroup_of_cycle_center_quotient` for the `CommGroup` instance. -/
```
Comment (b-mehta) [inline Mathlib/GroupTheory/SpecificGroups/Cyclic.lean:515]:
```suggestion
  Also see `commGroupOfCyclicCenterQuotient` for the `CommGroup` instance. -/
```
and this one too actually

**Pull request closed without merging**